README.md

```diff
@@ -1,13 +1,31 @@
 # Project Labyrinth
+This is a pair programming project made by Karin Nordkvist and Gabriella Bolin. We have built a Labyrinth Game UI that retrieves a list of actions from the backend, and present them as selectable directions for the player. 
 
-Replace this readme with your own information about your project.
+## The problem
+We started by making two reducers, one for the labyrinth data and one for the ui. The labyrinth reducer keeps the actions for fetching data from the API and for setting the input value to current username. 
```

Comment:
You should be super proud of the great styling you have done! The page is really pretty with cool animations! ❤️

Reply:
Completely agree